blurb/blurb.py

```diff
@@ -913,24 +915,19 @@ def add():
     os.close(handle)
     atexit.register(lambda : os.unlink(tmp_path))
 
-    def init_tmp_with_template():
-        with open(tmp_path, "wt", encoding="utf-8") as file:
-            # hack:
-            # my editor likes to strip trailing whitespace from lines.
-            # normally this is a good idea.  but in the case of the template
-            # it's unhelpful.
-            # so, manually ensure there's a space at the end of the bpo line.
-            text = template
-
-            bpo_line = ".. bpo:"
-            without_space = "\n" + bpo_line + "\n"
-            with_space = "\n" + bpo_line + " \n"
-            if without_space not in text:
-                sys.exit("Can't find BPO line to ensure there's a space on the end!")
-            text = text.replace(without_space, with_space)
-            file.write(text)
+    if bpo:
+        try:
+            int(bpo)
+        except ValueError:
+            error("blurb add --bpo argument " + repr(bpo) + " is not a valid integer!")
 
-    init_tmp_with_template()
+    if section:
+        if section not in sections:
+            error("blurb add --section argument " + repr(section) + " is not a valid section!")
```

Comment:
How about printing the valid sections when this fails?


```suggestion
            error("blurb add --section argument " + repr(section) + " is not a valid section! Use one of:\n" + "\n".join(sections))
```


For example:

```console
$ python blurb.py add --bpo 123 --section aaa
Error: blurb add --section argument 'aaa' is not a valid section! Use one of:
Security
Core and Builtins
Library
Documentation
Tests
Build
Windows
macOS
IDLE
Tools/Demos
C API
```